Documentation/edgefs-s3-crd.md

```diff
@@ -0,0 +1,142 @@
+---
+title: AWS S3
+weight: 36
+indent: true
+---
+
+# AWS S3
+
+Rook allows creation and customization of AWS S3 compatible services through the custom resource definitions (CRDs).
+The following settings are available for customization of S3 services.
+
+## Sample
+
+```yaml
+apiVersion: edgefs.rook.io/v1alpha1
+kind: S3
+metadata:
+  name: s301
+  namespace: rook-edgefs
+spec:
+  instances: 3
+  placement:
+  #  nodeAffinity:
```

Comment:
a lot of these examples have commented out placement/resources, but maybe some more of the EdgeFS specific fields may be more useful to the users

Reply:
perhaps in the future versions we will add more, but at the moment not that many options available except edgefs crd, which is well documented anyways. but yes, agree. we should add more to these..